examples/vrf_snmp_context_community/main.tf

```diff
@@ -1,22 +1,36 @@
 terraform {
-    required_providers {
-        aci = {
-        source = "ciscodevnet/aci"
-        }
+  required_providers {
+    aci = {
+      source = "ciscodevnet/aci"
     }
+  }
 }
 
 provider "aci" {
-    username = ""
-    password = ""
-    url      = ""
-    insecure = true
+  username = ""
+  password = ""
+  url      = ""
+  insecure = true
 }
 
 resource "aci_vrf_snmp_context_community" "example" {
-	vrf_snmp_context_dn = aci_vrf_snmp_context.test.id
-	name = "test"
-	description = "From Terraform"
-	annotation = "Test_Annotation"
-	name_alias = "Test_name_alias"
+  vrf_snmp_context_dn = aci_vrf_snmp_context.example.id
```

Comment:
Isn't this supposed to be -
parent_dn = aci_vrf_snmp_context.example.id

Reply:
In the new one yes but this is the old one and the old one has kept the same syntax as before.

Reply:
ok